plugin/src/main/java/com/playmonumenta/scriptedquests/listeners/PlayerListener.java

```diff
@@ -71,8 +89,44 @@ public void playerInteractEvent(PlayerInteractEvent event) {
 				mPlugin.mRaceManager.restartRaceByClick(player);
 			}
 		}
+
+
 	}
 
+	@EventHandler(priority = EventPriority.LOWEST)
+	public void playerAnimationEvent(PlayerAnimationEvent event) {
+
+		Player player = event.getPlayer();
+		//This only applies to players in adventure mode looking at blocks (not air)
+		if (player.getGameMode() != GameMode.ADVENTURE || player.getTargetBlock(null, 4).getType() == Material.AIR || event.getAnimationType() != PlayerAnimationType.ARM_SWING) {
```

Comment:
Can we swap the order here to move getAnimationType() check before the getTargetBlock? That is a much cheaper check, getTargetBlock is a raytrace.